### ISSUE TITLE
Mention in the docs that `Foo =>` is `{auto _ : Foo} ->`

### ISSUE THREAD

**Issue by joelberkeley** [idris-lang/Idris2#2626]
This is asked so many times in discord that I reckon it should be really obvious in the docs. I can't do that right now so am making an issue so it's not forgotten.

**Comment by andrevidela** [comment]
were you looking for this page? https://idris2.readthedocs.io/en/latest/updates/updates.html#auto-implicits-and-interfaces

**Comment by joelberkeley** [comment]
@andrevidela not really. I mean more obvious. "Changes since Idris 1" is quite a niche page

**Comment by andrevidela** [comment]
Indeed, the information is there but it seems everyone is missing it.

Maybe we could create a page "idris2 desugarings" in here? https://idris2.readthedocs.io/en/latest/reference/index.html

**Comment by joelberkeley** [comment]
I don't have a good enough grasp on the readthedocs to say (I'm currently down a rabbit hole of seeing if I can improve the README), but if we did that would there be a risk of proliferating duplication as desugaring appears in numerous parts of Idris?

**Comment by andrevidela** [comment]
Indeed, maybe it could fit in the FAQ? https://idris2.readthedocs.io/en/latest/faq/faq.html?highlight=FAQ

**Comment by joelberkeley** [comment]
could do. what about as a note in the main interfaces page?
